components/style/properties/longhand/ui.mako.rs

```diff
@@ -46,6 +46,21 @@ ${helpers.predefined_type("-moz-window-opacity", "Opacity", "1.0", products="gec
                           internal=True,
                           spec="None (Nonstandard internal property)")}
 
+${helpers.predefined_type("-moz-window-transform", "Transform", "Default::default()",
```

Comment:
Then this `Default::default()` can become `Transform::none()`, which I think it's clearer.

Reply:
Except you cannot. It needs to be `computed::Transform::none()`, but that's probably fine...

Reply:
Actually, that doesn't work either. You would need `generics::transform::Transform::none()` because associated items don't work with type alias IIRC.